packages/components/src/item-group/stories/index.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import { ItemGroup } from '../item-group/component';
+import { Item } from '../item/component';
+import type { ItemProps } from '../types';
+
+const meta: ComponentMeta< typeof ItemGroup > = {
+	component: ItemGroup,
+	title: 'Components (Experimental)/ItemGroup',
+	subcomponents: { Item },
+	argTypes: {
+		as: { control: { type: null } },
+		children: { control: { type: null } },
+	},
+	parameters: {
+		controls: { expanded: true },
+		docs: { source: { state: 'open' } },
+	},
+};
+export default meta;
+
+const mapPropsToItem = ( props: ItemProps, index: number ) => (
+	<Item { ...props } key={ index } />
+);
+
+const Template: ComponentStory< typeof ItemGroup > = ( props ) => (
+	<ItemGroup { ...props } />
+);
+
+export const Default: ComponentStory< typeof ItemGroup > = Template.bind( {} );
```

Comment:
Also, note that I removed the hardcoded `max-width` from the component, since we can now leverage some custom Storybook tools to change the container's width